app/service/ccd-service.ts

```diff
@@ -12,7 +12,8 @@ const logLabel: string = getLogLabel(__filename);
 
 export const Events = {
   EDIT_APPEAL: { id: 'editAppeal', summary: 'Update appeal case AIP', description: 'Update appeal case AIP' },
-  SUBMIT_APPEAL: { id: 'submitAppeal', summary: 'Submit Appeal case AIP', description: 'Submit Appeal case AIP' }
+  SUBMIT_APPEAL: { id: 'submitAppeal', summary: 'Submit Appeal case AIP', description: 'Submit Appeal case AIP' },
+  EDIT_APPELLANT_APPEAL_REASON: { id: 'editAppellantAppealReason', summary: 'Upload Reason for Appeal case-building AIP', description: 'Upload Reason for Appeal case-building AIP' }
```

Comment:
As far as I know, there are only two Events, edit and submit. Also, I don't see where are you using this event?